code/modules/reagents/reagent_containers/spray.dm

```diff
@@ -18,6 +18,11 @@
 
 
 /obj/item/weapon/reagent_containers/spray/afterattack(atom/A as mob|obj, mob/user)
+	if(user.disabilities && CLUWNEMUT && prob(40))
```

Comment:
I'm wondering, what if there's a (even smaller) chance you spray yourself with the bottle?